draft-pauly-quic-datagram.md

```diff
@@ -171,6 +182,14 @@ datagrams within a single QUIC connection, as the Stream ID does for
 STREAM frames. Demultiplexing datagram data is the responsibility of the
 application.
 
+## Datagram Identifiers {#datagram-id}
+
+Since several applications relying on datagrams have the need to identify which
+application-level flow a given datagram is a part of, DATAGRAM frames carry a
+datagram identifier. Applications that do not have a need for the identifier
+can use the value zero on their DATAGRAM frames and use the DATAGRAM_ID bit
+to omit sending the identifier over the wire.
```

Comment:
Should we add text to say that if a mix of DATAGRAM+ID frames are used with DATAGRAM+NO_ID frames, that the non-ID frames are implicitly part of the "0" flow?

Reply:
I had been thinking this as well, since that makes the application interface interaction really nice and clean.